index.js

```diff
@@ -100,7 +100,7 @@ const generateTokensFiles = async (tokens) => {
   directoryGuard(DEFAULT_IMAGES_PATH);
   for (let token of tokens) {
     generateTokenMetadata(token);
-    await generateTokenImage(token);
+    await  generateTokenImage(token);
```

Comment:
Could you remove this extra blank space?

Reply:
all cleaned up. welcome home, hope you had a good vacation, thanks for the repo